services/web/server/src/simcore_service_webserver/login/handlers_confirmation.py

```diff
@@ -71,8 +71,10 @@ def _parse_extra_credits_in_usd_or_none(
     confirmation: ConfirmationTokenDict,
 ) -> PositiveInt | None:
     with suppress(ValidationError, JSONDecodeError):
-        invitation = InvitationData.parse_raw(confirmation.get("data", "EMPTY"))
-        return invitation.extra_credits_in_usd
+        confirmation_data: str | None = confirmation.get("data", "EMPTY")
```

Comment:
This is now more convoluted. The idea here is to raise `ValudationError`if something goes wrong and then suppressed. 
if confirmation_data is None, then parse_raw will raise. There is no need for this check